narratives/ncov-19B_resurgence.md

```diff
@@ -0,0 +1,192 @@
+
+19B:501T contains the spike 69-70 deletion present in 501Y.V1.
+This lineage has been sampled in France, Switzerland, Jordan, Turkey, Egypt, the United Arab Emirates, and the United States.
+
+19B:501Y has several additional spike mutations.
+L18F is present in 501Y.V2 and 501Y.V3.
+A [preprint by McCallum et al](https://www.biorxiv.org/content/10.1101/2021.01.14.426475v1), found that this mutation reduced binding by one of six tested neutralizing N-terminal domain antibodies derived from COVID-19 convalescent serum.
+
+The lineage also contains spike A653V, D796Y, G1219V, and L452R.
+We will discuss L452R emergences later in this report.
+
+19B:501Y has been sampled in Mayotte, Turkey, England, France, the Netherlands, Switzerland, and Denmark.
+
+<!-- There is NO right-side text on this slide -->
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- SLIDE 6 -->
+# [E484K present in 19B](https://nextstrain.org/groups/blab/ncov/19B?branchLabel=aa&c=gt-S_484&d=tree&f_clade_membership=19B&gt=S.613H&label=clade:19B&legend=open)
+
+<!-- This is the left-side text -->
+Here we are highlighting another subclade in 19B with multiple spike mutations.
+While this clade does not have a D614G substitution, it instead has a substitution in the adjacent residue, Q613H.
+
+The subclade also contains F157L, V367F, and P681R.
+P681H, a different but still positively charged amino acid at the same location, is present in 501Y.V1.
+In a deep mutational scanning experiment of the receptor binding domain by [Starr et al](https://jbloomlab.github.io/SARS-CoV-2-RBD_DMS/), V367F was associated with increased expression.
+
+E484K (yellow) emerged within this subclade.
+This mutation is present in 501Y.V2 and 501Y.V3.
+Multiple preprints have demonstrated reduced neutralization to convalescent sera, monoclonal antibodies and sera from individuals with Moderna or Pfizer-BioNTech vaccines with this mutation ([Greaney et al](https://www.biorxiv.org/content/10.1101/2020.12.31.425021v1), [Liu et al](https://www.biorxiv.org/content/10.1101/2020.11.06.372037v1), and [Wang et al](https://www.biorxiv.org/content/10.1101/2020.12.31.425021v1)).
+
+R190S, a mutation present in 501Y.V3, occurs on the branch leading up to E484K.
+
+<!-- There is NO right-side text on this slide -->
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- SLIDE 7 -->
+# [Multiple emergences of L452R](https://nextstrain.org/groups/blab/ncov/19B?c=gt-S_452&d=tree&f_clade_membership=19B&p=full&label=clade:19B&legend=open)
+<!-- This is the left-side text -->
+Coloring by amino acid at spike 452, we observe three independent emergences of 452R mutations (blue) and one emergence of a 452M mutation (yellow). L452R has been observed in a [lineage growing in frequency in California]((https://www.cdph.ca.gov/Programs/OPA/Pages/NR21-020.aspx) but the biological impact of this mutation, if any, is unknown.
+
+One emergence of 452R was in 19B:501Y on a 614D backbone.
```

Comment:
```suggestion
One emergence of 452R is in 19B:501Y on a 614D backbone.
```
This is talking about the current tree, right? The past tense leaves me a little confused if you're talking about a published study?